subsys_display_view.py

```diff
@@ -59,7 +59,7 @@ def stop(cls):
         pass
 
     @classmethod
-    def run(cls, frame, **karg):
+    def run(cls, frame: numpy.ndarray, **karg):
```

Comment:
I believe this should be either : 
*args or **kwargs
**karg is something i never saw before that